artifacts/definitions/Windows/Applications/MegaSync.yaml

```diff
@@ -0,0 +1,84 @@
+name: Windows.Application.MegaSync
+description: |
+  This artifact will parse MEGASync logs and enables using regex to search for 
+  entries of interest.
+  
+  With UploadLogs selected a copy of the logs are uploaded to the server.
+  SearchVSS enables search over VSS and dedup support.
+  
+author: "Matt Green - @mgreen27"
+
+reference:
+  - https://attack.mitre.org/techniques/T1567/002/
+
+precondition: SELECT OS From info() where OS = 'windows'
+
+parameters:
+  - name: LogFiles
+    default: 'C:\Users\*\AppData\Local\Mega Limited\MEGAsync\logs\*.log'
+  - name: SearchRegex
+    description: "Regex of strings to search in line."
+    default: 'Transfer\s\(UPLOAD\)|upload\squeue|local\sfile\saddition\sdetected|Sync\s-\ssending\sfile|\"user\"'
+  - name: WhitelistRegex
+    description: "Regex of strings to leave out of output."
+    default:
+  - name: SearchVSS
+    description: "Add VSS into query."
+    type: bool
+  - name: UploadLogs
+    description: "Upload MEGASync logs."
+    type: bool
+    
+    
+sources:
+  - query: |
+      -- Find target files
+      LET files = SELECT *
+        FROM if(condition=SearchVSS,
+            then= {
+                SELECT *
+                FROM Artifact.Windows.Search.VSS(SearchFilesGlob=LogFiles)
+            },
+            else= {
+                SELECT *, FullPath as Source
+                FROM glob(globs=LogFiles)
+            })
+
+      -- Collect all Lines in scope of regex search
+      LET output <= SELECT * FROM foreach(row=files,
+          query={
+              SELECT Line, FullPath,
+                Mtime,
+                Atime,
+                Ctime,
+                Size
+              FROM parse_lines(filename=FullPath,accessor='file')
+              WHERE TRUE
```

Comment:
It is a bit redundant to have that TRUE there?

Reply:
@scudette yes it is.

I added this as an idea to allow consistent WHERE clauses when copying and pasting a lot. Its something I do regularly when writing VQL. The idea means you add TRUE on the first line then can simple copy and paste AND, or AND NOT components that are useful depending on use case. This one snuck into this PR but we can remove it.